drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -1834,15 +1834,15 @@ void RasterizerSceneGLES3::_render_geometry(RenderList::Element *e) {
 
 void RasterizerSceneGLES3::_setup_light(RenderList::Element *e, const Transform &p_view_transform) {
 
-	int omni_indices[16];
+	int maxobj = state.max_forward_lights_per_object;
+
+	int *omni_indices = (int *)memalloc(maxobj * sizeof(int));
```

Comment:
alloca would be better here for stack allocation.

Reply:
I made the change, thanks for the suggestion.

Reply:
This makes MSVC unhappy:
```
 cl /Fodrivers\gles3\rasterizer_scene_gles3.windows.opt.tools.64.obj /c drivers\gles3\rasterizer_scene_gles3.cpp /TP /std:c++14 /nologo /O2 /Z7 /MT /Gd /GR /utf-8 /W3 /wd4267 /wd4244 /wd4305 /wd4018 /wd4800 /EHsc /WX /DDEBUG_ENABLED /DWINDOWS_ENABLED /DOPENGL_ENABLED /DWASAPI_ENABLED /DWINMIDI_ENABLED /DTYPED_METHOD_BIND /DWIN32 /DMSVC /DWINVER=0x0601 /D_WIN32_WINNT=0x0601 /DNOMINMAX /D_WIN64 /DPTRCALL_ENABLED /DTOOLS_ENABLED /DGDSCRIPT_ENABLED /DMINIZIP_ENABLED /DZSTD_STATIC_LINKING_ONLY /DGLAD_ENABLED /DGLES_OVER_GL /DFREETYPE_ENABLED /DSVG_ENABLED /Ithirdparty\nanosvg /Ithirdparty\freetype\include /Ithirdparty\libpng /Ithirdparty\glad /Ithirdparty\zstd /Ithirdparty\zlib /Iplatform\windows /I.
rasterizer_scene_gles3.cpp
D:\a\godot\godot\drivers\gles3\rasterizer_scene_gles3.cpp(2222) : error C2220: the following warning is treated as an error
D:\a\godot\godot\drivers\gles3\rasterizer_scene_gles3.cpp(2222) : warning C4750: 'void __cdecl RasterizerSceneGLES3::_setup_light(struct RasterizerSceneGLES3::RenderList::Element * __ptr64,class Transform const & __ptr64) __ptr64': function with _alloca() inlined into a loop
```